<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>STMessageCenterClass</class>
 <widget class="QDialog" name="STMessageCenterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>STMessageCenter</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {background-color: #ffffff;border: 1px solid #e3e3e3;}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widTitle" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #434555;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>消息中心</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>448</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_17" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pbClose">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">
QPushButton{border-image: url(:/SoftTerminal/images/close.png);}

QPushButton:hover:!pressed {
	
	background-color: #ff6741;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QTabWidget{background-color: #ffffff;}
QTabWidget::pane{border:0px;}

/*QTabWidget::tab-bar{alignment:center;}*/

#tabWidget QTabBar::tab
{
    min-width: 75px;
    min-height: 28px;
	font: 10pt &quot;微软雅黑&quot;;
}


#tabWidget QTabBar::tab:selected
{
    border-bottom: 2px solid #00c36e;
	color:#00c36e;
}

#tabWidget QTabBar::tab:!selected
{
    border-bottom: 0px;
	color: rgb(67, 69, 85);
}
</string>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
        <property name="usesScrollButtons">
         <bool>false</bool>
        </property>
        <property name="tabsClosable">
         <bool>false</bool>
        </property>
        <property name="tabBarAutoHide">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="twNewFriend">
         <attribute name="title">
          <string>好友请求</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignTop">
           <widget class="QWidget" name="widNewFriendMessage" native="true">
            <property name="styleSheet">
             <string notr="true">QWidget#widMessage{background-color: rgb(255, 255, 255);}</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/softterminal.qrc"/>
 </resources>
 <connections/>
</ui>
